### ISSUE TITLE
DatePicker and DateInput Accessibility issues

### ISSUE THREAD

**Issue by kspeyanski** [telerik/kendo-react#918]
The `aria-expanded` attribute is [not allowed](https://www.w3.org/TR/wai-aria-1.1/#spinbutton) on the `role="spinbutton"` we currently have on the `DateInput` component.

Possible fixes to this might be moving the `aria-expanded` up in the tree of component (e.g on the wrapping component of the `DateRangePicker`), but further research is required.

**Comment by filipKovachev** [comment]
DatePicker component has the following accessibility issue: ARIA attribute is not allowed: aria-expanded="false"
Ticket ID: 1546676

**Comment by filipKovachev** [comment]
DateInput component has the following accessibility issue: ARIA attribute: aria-label is not allowed
Ticket ID: 1546677

**Comment by WissamProgress** [comment]
Ticket ID: 1630662

When using a screen reader and navigating through the DateInput component using the left and right arrow keys, the values are not announced.

**Comment by iadnonov** [comment]
Not reproducible in version 7.0.2
